# HG changeset patch
# User Rong Bao <webmaster@csmantle.top>
# Date 1787653804 0
# Node ID 35379c0555997725944792d197bb45ec4ac8e263
# Parent  95a824e3f9c47440352f23af909a63999b6ade79
Bug 2065815 - Rename usage of macro Success to X11Success in nsXRemoteServer.cpp. r=mossop

This is a fixup to Bug 2065007.

Differential Revision: https://phabricator.services.mozilla.com/D321140


diff --git a/toolkit/components/remote/nsXRemoteServer.cpp b/toolkit/components/remote/nsXRemoteServer.cpp
--- a/toolkit/components/remote/nsXRemoteServer.cpp
+++ b/toolkit/components/remote/nsXRemoteServer.cpp
@@ -105,17 +105,17 @@ bool nsXRemoteServer::HandleNewProperty(
                                 &len,           /* nitems_return */
                                 &bytes_after,   /* bytes_after_return */
                                 (unsigned char**)&data); /* prop_return
                                                             (we only care
                                                             about the first ) */
 
     // Failed to get property off the window or
     // got a part only
-    if (result != Success || bytes_after != 0) {
+    if (result != X11Success || bytes_after != 0) {
       return false;
     }
 
     // Failed to get the data off the window or it was the wrong type?
     if (!data) {
       return false;
     }
 

